sql/src/main/java/io/crate/analyze/SetStatementAnalyzer.java

```diff
@@ -80,12 +82,51 @@ public static SetAnalyzedStatement analyze(SetStatement node) {
     }
 
     public static SetLicenseAnalyzedStatement analyze(SetLicenseStatement node) {
-        Map<String, List<Expression>> settings = new HashMap<>();
+        long expiryDateMs = 0L;
+        String issuedTo = null;
+        String signature = null;
+        boolean invalidType = false;
         for (Assignment assignment : node.assignments()) {
             String settingName = ExpressionToStringVisitor.convert(assignment.columnName(), Row.EMPTY);
-            settings.put(settingName, ImmutableList.of(assignment.expression()));
+            Expression expression = assignment.expression();
+            // todo: revisit
+            switch (settingName) {
+                case "expirydate":
+                    if (expression instanceof LongLiteral) {
+                        expiryDateMs = ((LongLiteral) expression).getValue();
```

Comment:
I think the analyzer should be limited to making sure the right `settingName`s are provided, but it shouldn't unwrap the expressions yet. That should be done later in the planner using the `SymbolEvaluator`. 

That's necessary to support statements like `set license  key = ?`. It must be possible to analyze this without having the value for `?`. The value will be available in the planner (That's the `Row params` argument to the `execute`)